data/autoyast_sle12sp2/bug-872532_ix64ph1069.xml

```diff
@@ -0,0 +1,1349 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update/standard</media_url>
+        <product>SuSE-Linux-Updates</product>
+        <product_dir>/</product_dir>
+        <name>Updates</name> 
+      </listentry>
+    </add_on_products>
+  </add-on>
+
+  <bootloader>
+    <global>
+      <append> ${extra_cmdline} console=ttyS0,115200 resume=/dev/disk/by-uuid/09428588-05c9-4d7c-9b21-243d9c460aca splash=silent quiet showopts crashkernel=164M-:82M</append>
+      <append_failsafe>showopts apm=off noresume edd=off powersaved=off nohz=off highres=off processor.max_cstate=1 nomodeset x11failsafe</append_failsafe>
+      <default>SUSE Linux Enterprise Server 12</default>
+      <gfxbackground>/boot/grub2/themes/SLE/background.png</gfxbackground>
+      <gfxmode>auto</gfxmode>
+      <gfxtheme>/boot/grub2/themes/SLE/theme.txt</gfxtheme>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>true</os_prober>
+      <suse_btrfs>true</suse_btrfs>
+      <timeout config:type="integer">8</timeout>
+      <xen_append>console=com1 com1=115200 </xen_append>
+      <xen_kernel_append> resume=/dev/disk/by-uuid/09428588-05c9-4d7c-9b21-243d9c460aca splash=silent quiet showopts</xen_kernel_append>
+    </global>
+    <loader_type>grub2</loader_type>
+    <sections config:type="list">
+      <section>
+        <append>${extra_cmdline}   console=ttyS0,115200 resume=/dev/disk/by-uuid/09428588-05c9-4d7c-9b21-243d9c460aca splash=silent quiet showopts crashkernel=164M-:82M</append>
```

Comment:
perhaps I don't get autoyast, but what is the purpose of hardcoding UUIDs and old kernel versions in here?